evalml/data_checks/ts_parameters_data_check.py

```diff
@@ -39,6 +39,36 @@ def validate(self, X, y=None):
         Returns:
             dict: dict with a DataCheckError if parameters are too big for the split sizes.
 
+        Examples:
```

Comment:
Realized there were no tests for `TimeSeriesParametersDataCheck` so I took this opportunity to add one